user/migrations/0009_auto_20200321_0441.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-03-21 11:41
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('user', '0008_user_can_review_dubious'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='user',
+            name='is_organizer',
+        ),
+        migrations.RemoveField(
+            model_name='user',
+            name='is_volunteer',
+        ),
+        migrations.AddField(
+            model_name='user',
+            name='type',
+            field=models.CharField(choices=[('H', 'Hacker'), ('M', 'Mentor'), ('S', 'Sponsor'), ('V', 'Volunteer'), ('O', 'Organizer')], default='H', max_length=2),
+        ),
```

Comment:
Could you change the name of the migration to something more named (instead of auto....)

Also, if you can, would it be possible to do use https://docs.djangoproject.com/en/3.0/ref/migration-operations/#runsql to update the DB from the previous to new state? (if `is_organizer` set type to `organizer`.....) Or https://docs.djangoproject.com/en/3.0/ref/migration-operations/#runpython

Reply:
Aren't migrations named automatically?

Reply:
Yes, but you can name them by using the `-n` option.

Also, mind to provide the `reverse_sql` (SQL to reverse the migration). Mostly for completeness (if someone migrates and has issues can rollback safely.